CRM/Core/DAO.php

```diff
@@ -888,7 +888,7 @@ public static function getAttribute($class, $fieldName = NULL) {
    *
    * @param array $record
    *
-   * @return $this
```

Comment:
using $this saves a lot of IDE messages

![image](https://user-images.githubusercontent.com/336308/118234217-2704c080-b4e7-11eb-9a98-95b808eb4a51.png)

Althought there is greyness in that thread & maybe self is better https://stackoverflow.com/questions/6945564/should-i-use-return-self-this-or-the-current-class

Reply:
Well, it's a static function, so technically it can't return `$this`. I hadn't tried `self` or `static` but if the IDE understands it, great.

Reply:
I've updated this to use `static` which PhpStorm understands and is more correct than `$this`.